cmd/neofs-cli/internal/key/raw.go

```diff
@@ -51,12 +60,23 @@ func GetOrGenerate(cmd *cobra.Command) *ecdsa.PrivateKey {
 }
 
 func getOrGenerate(cmd *cobra.Command) (*ecdsa.PrivateKey, error) {
-	if viper.GetBool(commonflags.GenerateKey) {
```

Comment:
```
Global Flags:
  -c, --config string   Config file (default is $HOME/.config/neofs-cli/config.yaml)
```
Maybe it's unwrapped into the path/pass already, I have not checked.

Reply:
User may have some config file, but still want to run a specific command in "anonymous" mode. From what I see now he'll just receive the key from `get()` without any possibility to override. That's the reason to keep `commonflags.GenerateKey` at the same time, it doesn't make much sense otherwise.

Reply:
done